### PR TITLE
add: `nautilus-open-in-blackbox-git`

### PR DESCRIPTION
**Pull request by Anifyuli**

### CODE REVIEW

File: packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name="nautilus-open-in-blackbox-git"
+gives="nautilus-open-in-blackbox"
+pkgver="0.1.1"
+pkgrel="1"
+arch='any'
```
Comment (vigress8) [inline packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript:5]:
```suggestion
```
Unnecessary

File: packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+url="https://github.com/ppvan/nautilus-open-in-blackbox.git"
+pkgdesc="Open current directory in BlackBox from Nautilus context menu"
+maintainer="Anifyuli <anifyuliansyah@gmail.com>"
+depends=("python3-nautilus")
+makedepends=('git')
+pkgver() {
+  git ls-remote "${url}" main | head -n1 | cut -f1 | cut -c1-8
+}
+package() {
+  cd "${name}-${pkgver}"
```
Comment (vigress8) [inline packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript:15]:
```suggestion
pkgver() {
  git ls-remote "${url}" main | cut -f1 | cut -c1-8
}
package() {
```

**Review by vigress8 — CHANGES_REQUESTED** [review]

File: packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript (hunk 1 above)
Comment (Anifyuli) [reply packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript]: Done

File: packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript (hunk 2 above)
Comment (Anifyuli) [reply packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript]: It's done too

File: packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name="nautilus-open-in-blackbox-git"
+gives="nautilus-open-in-blackbox"
+pkgver="0.1.1"
+url="https://github.com/ppvan/nautilus-open-in-blackbox.git"
+pkgdesc="Open current directory in BlackBox from Nautilus context menu"
+maintainer="Anifyuli <anifyuliansyah@gmail.com>"
+depends=("python3-nautilus")
+pkgver() {
+  git ls-remote "${url}" main | head -n1 | cut -f1 | cut -c1-8
+}
```
Comment (Elsie19) [inline packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript:10]:
```suggestion
}

```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (Elsie19) [inline packages/nautilus-open-in-blackbox-git/nautilus-open-in-blackbox-git.pacscript:11]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**